src/UnitTestProject1/UnitTestProject1.csproj

```diff
@@ -50,14 +50,10 @@
     </Otherwise>
   </Choose>
   <ItemGroup>
-    <Compile Include="AlphaConvertionTests.cs" />
     <Compile Include="ColorOperationTests.cs" />
     <Compile Include="CmykColorTests.cs" />
-    <Compile Include="HexTests.cs" />
     <Compile Include="HsvColorTests.cs" />
     <Compile Include="CmyColorTests.cs" />
-    <Compile Include="LessTests.cs" />
-    <Compile Include="ParseTests.cs" />
     <Compile Include="Samples\HtmlColorSample.cs" />
     <Compile Include="HslColorTests.cs" />
     <Compile Include="RgbColorTests.cs" />
```

Comment:
Seems like you might have had an older version of `src/UnitTestProject1/UnitTestProject1.csproj` that then got committed. Could you revert this part of the PR?

Reply:
Fixed